workflows/dynamic/index.ts

```diff
@@ -237,9 +239,11 @@ export default function (listener) {
           // Perform error handling, such as displaying an error message to the user or triggering a fallback behavior
         }
 
-        await api.jobs.complete(jobId, {
-          info: 'Data synced to the HCM.show app.',
-        });
+        if (callback === '{"success":true}') {
```

Comment:
Got stuck on this and not sure why this a string version of the object is necessary. 

SS below of the log for callback

<img width="237" alt="CleanShot 2023-07-18 at 14 23 09@2x" src="https://github.com/FlatFilers/hcm-show-config/assets/12041898/5bfdf42d-7537-402a-a521-118725e30117">

Reply:
This does not work as the conditional

```
if (callback && callback.success === true) {
          await api.jobs.complete(jobId, {
            info: 'Data synced to the HCM.show app.',
          });
        }
```

Reply:
Sooo I see now the backend serializes the object into a JSON string. 😞

Reply:
Editing now to parse it, so it doesnt look so ugly